### PR TITLE
Add SVGDOM limitations and instructions for adding assets and fonts

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] - Docs: Instructions on adding static assets
- [Add] - Docs: Experimental instructions on adding fonts
- [Add] - Docs: Limitations to SVGDOM API

### CODE REVIEW

File: docs/static-diagram-core.md [hunk 1]
```diff
@@ -73,11 +93,35 @@ When choosing a package, a version targeted for browsers may be available and mo
...
+    SVG_ROOT!.querySelector("#logo-container")?.appendChild(logo_node.node);
+}
+```
+
 ## Usage
 
 Unlike the simplified server render function, SVGDOM is bundled to allow DOM manipulation mirroring the DOM API found in the browser.
 
-We add some boilerplate at top of [src/static/render.ts](https://github.com/ClearCalcs/custom-diagram-boilerplate/blob/main/src/static/render.ts) called by the `render()`. The author should use the SVGDOM primitives for `document` and `window` in place of the browser equivalents and generate an SVG string from the result of those object's manipulation. We have excluded the fontkit boilerplate for clarity, however where text elements are used this should be included.
+We add some boilerplate at top of `render.ts` called by the `render()`. The author should use the SVGDOM primitives for `document` and `window` in place of the browser equivalents and generate an SVG string from the result of those object's manipulation.
```
Comment (JayserBeam) [inline docs/static-diagram-core.md:124]: This looks like it picked this over the existing changes but these were changed in another PR?

**Review by JayserBeam — CHANGES_REQUESTED** [review]
Think it's picked the wrong route to take on a rebase, other than that, readable, understandable

**Comment by michaellisitsa** [comment]
@JayserBeam ready for re-review

**Review by JayserBeam — APPROVED** [review]
There's now a double blank line but its [ultra-nit-pick] so i'm not insanely concerned

**Pull request merged**